src/website/app/demos/Popover/bestPractices.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { createStyledComponent } from '../../../../utils';
+import Button from '../../../../Button';
+import Heading from '../../Heading';
+import Popover from '../../../../Popover';
+
+const styles = {
+  badDemo: {
+    width: '50em'
+  },
+  demoLayout: {
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  goodDemo: { width: '15em' },
+  p: ({ theme }) => ({
+    marginBottom: 0,
+    marginTop: theme.spacing_single
+  })
+};
+
+const GoodDemo = createStyledComponent('div', styles.goodDemo);
+const DemoLayout = createStyledComponent('div', styles.demoLayout);
+const P = createStyledComponent('p', styles.p);
+
+class PopOverDo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { open: false };
+  }
+
+  render() {
+    const DemoContent = (
+      <GoodDemo>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec mattis
+          pretium massa. Aliquam erat volutpat. Nulla facilisi. Donec vulputate
+          interdum sollicitudin. Nunc lacinia auctor quam sed pellentesque.
+        </p>
+      </GoodDemo>
+    );
+    return (
+      <DemoLayout>
+        <Button
+          primary
+          onClick={() => this.setState({ open: !this.state.open })}>
+          Learn more
+        </Button>
+        <Popover isOpen={this.state.open} content={DemoContent}>
+          <P>Mineral UI</P>
+        </Popover>
+      </DemoLayout>
+    );
+  }
+}
+
+const BadContent = (
```

Comment:
😆 Whoa!

I don't have anything constructive to add. It just made me laugh when I triggered this one.